geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionRedundancyTracker.java

```diff
@@ -131,7 +136,13 @@ synchronized void decrementNoCopiesBucketCount() {
     }
   }
 
+  public int getActualRedundancy() {
+    return actualRedundancy;
+  }
+
   void setActualRedundancy(int actualRedundancy) {
-    stats.setActualRedundantCopies(Math.max(actualRedundancy, 0));
+    int nonNegativeRedundancy = Math.max(actualRedundancy, 0);
+    this.actualRedundancy = nonNegativeRedundancy;
```

Comment:
Minor redundant variable assignment could be removed.

Reply:
I did it for readibility.  in my head there were three things going on here, 1) check if the variable is negative, and if so make it 0, 2) assign it to the internal field, and 3) send it over to stats, so I kept each of those on its own line.

But I was curious if this had a performance impact, so I made a microbenchmark!!!  https://gist.github.com/prettyClouds/71caff788a27f49da204e964afd3b463

and the results for me were: 
```
Benchmark                           Mode  Cnt           Score   Error  Units
FunBenchmarkRename.extra_variable  thrpt        294099676.492          ops/s
FunBenchmarkRename.no_variable     thrpt        290787344.255          ops/s
FunBenchmarkRename.noop            thrpt       3909999896.655          ops/s
```

no significant difference...and its an int so shouldn't affect garbage collection.

check out the benchmark and let me know if you think this makes sense.

Reply:
Yes, it purely readability preference, the JIT will eliminate it. I am fine keeping it in. Way to go using JMH to test out things.